generated/niscope/niscope.proto

```diff
@@ -979,10 +966,7 @@ message CheckAttributeViInt64Request {
   nidevice_grpc.Session vi = 1;
   string channel_list = 2;
   NiScopeAttribute attribute_id = 3;
-  oneof value_enum {
```

Comment:
Looking at this again I'm concerned with the oneof going away and value_raw's field number is changing from 5 to 4. We discussed the cause of this in the metadata files but summarizing this is because initially for grpc-device we had incorrectly marked the `P2P_NOTIFY_MESSAGE_PUSH_ADDR` attribute with the `'enum': 'AddressType'` tag which in turn causes `NiScopeInt64AttributeValues` to get generated and used here and in the `Set...` method. If we correctly remove that 'enum' tag now, we'll get this breaking change.

@harsha-bhushan , is this an acceptable breaking change? Or would we rather want to try to keep backwards compatibility here by still providing the `NiScopeInt64AttributeValues` enum input option (even if the values are kind of bogus) along with the `value_raw` option.

Reply:
Looking at the P2P_NOTIFY_MESSAGE_PUSH_ADDR attribute documentation, this specifies to which peer address a message should be pushed. It is unlikely someone would have used an enum for this & most likely they would have set a raw value. So, although it is a breaking change the impact may not be big, but rather beneficial? If someone was using the enum by mistake (which was incorrect), they will now be forced to use the raw value. Which seems the right thing to do.
(please correct me if i'm missing something)